azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -462,15 +465,24 @@ public void prefetchToken(final File tokenFile, final Props props, final Logger
     doPrefetch(tokenFile, props, logger, userToProxy);
   }
 
+  private String kerberosSuffix(final Props props) {
```

Comment:
It would help to add a comment with an example of the suffix string.

Reply:
Will do.